multi_host_migration/tests/migration_multi_host_timedrift.py

```diff
@@ -69,6 +70,8 @@ def check_diff(self, mig_data):
                     check_clocksource_cmd = params.get("check_clocksource_cmd")
                     clocksource = params.get("clocksource", "kvm-clock")
                     current_clocksource = session.cmd(check_clocksource_cmd)
+                    current_clocksource = re.findall(clocksource, current_clocksource)
+                    current_clocksource = "".join(current_clocksource)
```

Comment:
the output is not "kvm-clock", it contains redundant characters (such as "crtl"), need to remove them